src/ui/Range.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+    export let items = []
+    export let hasAllWithLabel;
+    export let onChange = (index, value) => {}
+    let className = ''
+    export {className as class}
+    export let style;
+
+    let index = 0;
+    $: if (hasAllWithLabel) items = [...items, hasAllWithLabel]
+
+    function clickHandler() {
+        index += 1;
+        if (index + 1 > items.length) {
+            index = 0
+        }
+        onChange(index, items[index])
```

Comment:
Pass item as the first argument, since it's more likely to be a used later by the library user